assets/ios/swift/Icons.xcassets/iconThread.imageset/Contents.json

```
@@ -0,0 +1 @@
+{"images":[{"filename":"iconThread.svg","idiom":"universal"}],"info":{"author":"xcode","version":1},"properties":{"preserves-vector-representation":true}}
```

Would it be possible to include the render mode property too please? It saves us needing to apply the render mode modifier on every use of the icon :)
```suggestion
{"images":[{"filename":"iconThread.svg","idiom":"universal"}],"info":{"author":"xcode","version":1},"properties":{"preserves-vector-representation":true,"template-rendering-intent":"template"}}
```